core/src/main/scala/kafka/utils/Time.scala

```diff
@@ -44,6 +46,8 @@ trait Time {
 
   def nanoseconds: Long
 
+  def hiResClockMs: Long = TimeUnit.NANOSECONDS.toMillis(nanoseconds)
```

Comment:
Thank you @gwenshap, @ijuma.
@ijuma Couldn't think of a reason for a different implementation in MockTime, but happy to move it if you think it will be useful.